### DISCUSSION TITLE
Can’t navigate in text editors with arrow keys when using an external keyboard

### DISCUSSION THREAD

**Discussion by nre-ableton** [community/community#12202] [Mobile]
When using an external keyboard with an iOS device, moving the cursor with the arrow keys in text editors (like to make comments, new issues, etc). doesn't work. Likewise, alt+arrow key to jump around words doesn't work either.

**Comment by stevepopovich** [comment]
Hello, again Nik 🙃 Thanks for the feedback, interesting use case. Passing this along to the iOS team!

**Reply by nre-ableton** [reply]
Also, I should note, upon further testing, the feature _sometimes_ works. But not all the time. I'm not sure what UI events cause it to break or become fixed again. Regardless, it's unreliable as currently implemented. Thanks for adding it to the backlog!

**Reply by stevepopovich** [reply]
oh, interesting. Thanks for the context!
